src/components/report/report.tsx

```diff
@@ -70,61 +65,90 @@ export class Report extends React.PureComponent<
     model_allocation: [],
     owner_allocation: [],
     vendor_allocation: [],
-    state_loaded: false,
+    stateLoaded: false,
     selectedTab: "global",
-    datacenter_loaded: false,
-    datacenterSelectionAlert: false,
-    no_data: false,
```

Comment:
the state loading variables were redundant so I minimized it and updated to get working for no data responses